cpu/arm7tdmi_gba/stdio_fb/stdio_fb.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2023 Bennet Blischke <bennet.blischke@haw-hamburg.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_gba_stdio_fb
+ * @{
+ *
+ * @file
+ * @brief       STDIO framebuffer driver
+ *
+ * This file provides a framebuffer driver for STDIO, implementing stdout only.
+ *
+ * @author      Bennet Blischke <bennet.blischke@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include "stdio_base.h"
+#include "periph_cpu.h"
+
+#include <string.h>
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+static const unsigned char systemFont[] =
+{
+#include "font_terminal.h"
+};
+
+#define BLACK 0x0000
+#define WHITE 0x7FFF
+
+static void clearScreen(void)
+{
+    memset(GBA_VRAM, 0x00, 240 * 160 * 2);
```

Comment:
Let's use a define for width and height, we can likely reuse this code for other displays.

Why the `* 2`? Is this double-buffering or do we have 16 bit/pixel?

(I don't see any flip buffer / commit command - how (often) is the framebuffer sent to the display?)

Reply:
I added the define for the screen size, got lost during refactoring oops. 
The `* 2` is indeed because of `15 bit/pixel` yes, `15` - at least in the display-mode selected for `stdio_fb`. The graphic options/settings/modes are impressive for such a little device. When a proper "gpu driver" is added in the future (not in this PR), this will probably get a rework. I hence just added a comment for now.